[2022-07-07 07:47:45] [[31merror[39m] FUNC: event transfer 
[2022-07-07 07:49:43] [[31merror[39m] FUNC: event transfer 
[2022-07-07 08:12:57] [[31merror[39m] Error: Key derivation failed - possibly wrong password
    at Accounts.decrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:438:15)
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:22:53)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 08:14:03] [[31merror[39m] Error: Key derivation failed - possibly wrong password
    at Accounts.decrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:438:15)
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:22:53)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 09:47:25] [[31merror[39m] TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined
    at Function.from (buffer.js:330:9)
    at Accounts.encrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:466:47)
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:90:48)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 09:47:49] [[31merror[39m] TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined
    at Function.from (buffer.js:330:9)
    at Accounts.encrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:466:47)
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:90:48)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:15:54] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:20:45] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:23:16] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:23:52] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:25:44] [[31merror[39m] TypeError: Cannot read property 'msg' of undefined
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:53)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:26:05] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:26:24] [[31merror[39m] register: Error: Missing where attribute in the options parameter passed to findOrCreate. Please note that the API has changed, and is now options only (an object with where, defaults keys, transaction etc.)
[2022-07-07 10:26:24] [[31merror[39m] Error: Missing where attribute in the options parameter passed to findOrCreate. Please note that the API has changed, and is now options only (an object with where, defaults keys, transaction etc.)
    at Function.findOrCreate (/home/trongnm/Documents/apieth/node_modules/sequelize/lib/model.js:1370:13)
    at Object.register (/home/trongnm/Documents/apieth/app/services/authServices.js:19:24)
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:89:47)
[2022-07-07 10:50:20] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:51:10] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:53:19] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:53:55] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:55:07] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:55:25] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 10:57:52] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:72:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:04:44] [[31merror[39m] register: Error: Missing where attribute in the options parameter passed to findOrCreate. Please note that the API has changed, and is now options only (an object with where, defaults keys, transaction etc.)
[2022-07-07 11:04:44] [[31merror[39m] Error: Missing where attribute in the options parameter passed to findOrCreate. Please note that the API has changed, and is now options only (an object with where, defaults keys, transaction etc.)
    at Function.findOrCreate (/home/trongnm/Documents/apieth/node_modules/sequelize/lib/model.js:1370:13)
    at Object.register (/home/trongnm/Documents/apieth/app/services/authServices.js:19:24)
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:89:47)
[2022-07-07 11:08:21] [[31merror[39m] TypeError: Cannot read property 'result' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:45:67)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:08:58] [[31merror[39m] TypeError: Cannot read property 'result' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:45:67)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:11:11] [[31merror[39m] TypeError: Cannot read property 'password' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:45:74)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:11:34] [[31merror[39m] TypeError: Cannot read property 'password' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:46:74)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:20:00] [[31merror[39m] TypeError: Cannot read property 'password' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:46:74)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:20:30] [[31merror[39m] TypeError: Cannot read property 'email' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:48:76)
[2022-07-07 11:20:53] [[31merror[39m] TypeError: Cannot read property 'id' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:48:101)
[2022-07-07 11:21:06] [[31merror[39m] TypeError: Cannot read property 'passwordWallet' of undefined
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:52:53)
[2022-07-07 11:21:37] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:14] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:24] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:25] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:25] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:26] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:29] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:22:36] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at register (/home/trongnm/Documents/apieth/app/controllers/authController.js:73:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:23:47] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:24:25] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:24:26] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:24:27] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at login (/home/trongnm/Documents/apieth/app/controllers/authController.js:37:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:45:01] [[31merror[39m] ReferenceError: validationResult is not defined
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:15:28)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:45:54] [[31merror[39m] TypeError: Cannot read property 'keystore' of undefined
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:29:46)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:46:11] [[31merror[39m] ReferenceError: retry is not defined
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:32:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:49:32] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:18:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:49:34] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:18:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 11:49:34] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at create (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:18:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:49:16] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:51:28] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:52:07] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:52:25] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:52:26] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:52:49] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:52:51] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:54:06] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 13:56:34] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 14:07:08] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at getBalance (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:66:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 14:30:06] [[31merror[39m] TypeError: Cannot read property 'keystore' of undefined
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:103:46)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 14:30:42] [[31merror[39m] TypeError: Cannot read property 'keystore' of undefined
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:103:46)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 14:31:12] [[31merror[39m] Error: Key derivation failed - possibly wrong password
    at Accounts.decrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:438:15)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:104:53)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 14:31:59] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:31:59] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:36:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:32:22] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:32:22] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:36:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:32:45] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:32:45] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:36:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:33:30] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:33:30] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:37:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:37:27] [[31merror[39m] Error: Returned error: insufficient funds for gas * price + value
    at Object.ErrorResponse (/home/trongnm/Documents/apieth/node_modules/web3-core-helpers/lib/errors.js:28:19)
    at /home/trongnm/Documents/apieth/node_modules/web3-core-requestmanager/lib/index.js:300:36
    at XMLHttpRequest.request.onreadystatechange (/home/trongnm/Documents/apieth/node_modules/web3-providers-http/lib/index.js:98:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:22)
    at XMLHttpRequest._setReadyState (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:208:14)
    at XMLHttpRequest._onHttpResponseEnd (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:318:14)
    at IncomingMessage.<anonymous> (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:289:61)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[2022-07-07 14:37:28] [[31merror[39m] Error: Returned error: insufficient funds for gas * price + value
    at Object.ErrorResponse (/home/trongnm/Documents/apieth/node_modules/web3-core-helpers/lib/errors.js:28:19)
    at /home/trongnm/Documents/apieth/node_modules/web3-core-requestmanager/lib/index.js:300:36
    at XMLHttpRequest.request.onreadystatechange (/home/trongnm/Documents/apieth/node_modules/web3-providers-http/lib/index.js:98:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:22)
    at XMLHttpRequest._setReadyState (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:208:14)
    at XMLHttpRequest._onHttpResponseEnd (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:318:14)
    at IncomingMessage.<anonymous> (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:289:61)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[2022-07-07 14:37:29] [[31merror[39m] Error: Returned error: insufficient funds for gas * price + value
    at Object.ErrorResponse (/home/trongnm/Documents/apieth/node_modules/web3-core-helpers/lib/errors.js:28:19)
    at /home/trongnm/Documents/apieth/node_modules/web3-core-requestmanager/lib/index.js:300:36
    at XMLHttpRequest.request.onreadystatechange (/home/trongnm/Documents/apieth/node_modules/web3-providers-http/lib/index.js:98:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:22)
    at XMLHttpRequest._setReadyState (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:208:14)
    at XMLHttpRequest._onHttpResponseEnd (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:318:14)
    at IncomingMessage.<anonymous> (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:289:61)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[2022-07-07 14:37:29] [[31merror[39m] Error: Returned error: insufficient funds for gas * price + value
    at Object.ErrorResponse (/home/trongnm/Documents/apieth/node_modules/web3-core-helpers/lib/errors.js:28:19)
    at /home/trongnm/Documents/apieth/node_modules/web3-core-requestmanager/lib/index.js:300:36
    at XMLHttpRequest.request.onreadystatechange (/home/trongnm/Documents/apieth/node_modules/web3-providers-http/lib/index.js:98:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:22)
    at XMLHttpRequest._setReadyState (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:208:14)
    at XMLHttpRequest._onHttpResponseEnd (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:318:14)
    at IncomingMessage.<anonymous> (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:289:61)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[2022-07-07 14:37:30] [[31merror[39m] Error: Returned error: insufficient funds for gas * price + value
    at Object.ErrorResponse (/home/trongnm/Documents/apieth/node_modules/web3-core-helpers/lib/errors.js:28:19)
    at /home/trongnm/Documents/apieth/node_modules/web3-core-requestmanager/lib/index.js:300:36
    at XMLHttpRequest.request.onreadystatechange (/home/trongnm/Documents/apieth/node_modules/web3-providers-http/lib/index.js:98:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request-event-target.js:34:22)
    at XMLHttpRequest._setReadyState (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:208:14)
    at XMLHttpRequest._onHttpResponseEnd (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:318:14)
    at IncomingMessage.<anonymous> (/home/trongnm/Documents/apieth/node_modules/xhr2-cookies/dist/xml-http-request.js:289:61)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[2022-07-07 14:45:03] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:45:03] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1000000000000000000, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:37:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:48:39] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:48:39] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:37:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:48:39] [[31merror[39m] transfer token: Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
[2022-07-07 14:48:39] [[31merror[39m] Error: overflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-overflow ] (fault="overflow", operation="BigNumber.from", value=1e+36, code=NUMERIC_FAULT, version=bignumber/5.6.2)
    at Logger.makeError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:233:21)
    at Logger.throwError (/home/trongnm/Documents/apieth/node_modules/@ethersproject/logger/lib/index.js:242:20)
    at throwFault (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:303:19)
    at Function.BigNumber.from (/home/trongnm/Documents/apieth/node_modules/@ethersproject/bignumber/lib/bignumber.js:206:17)
    at NumberCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/number.js:36:39)
    at /home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:74:19
    at Array.forEach (<anonymous>)
    at pack (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/array.js:60:12)
    at TupleCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/coders/tuple.js:71:33)
    at AbiCoder.encode (/home/trongnm/Documents/apieth/node_modules/@ethersproject/abi/lib/abi-coder.js:91:15)
    at ABICoder.encodeParameters (/home/trongnm/Documents/apieth/node_modules/web3-eth-abi/lib/index.js:127:27)
    at /home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:462:20
    at Array.map (<anonymous>)
    at Object._encodeMethodABI (/home/trongnm/Documents/apieth/node_modules/web3-eth-contract/lib/index.js:461:8)
    at Object.transfer (/home/trongnm/Documents/apieth/app/services/tokenServices.js:37:73)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:111:48)
[2022-07-07 14:53:24] [[31merror[39m] ReferenceError: BigNumber is not defined
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:99:29)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 15:11:52] [[31merror[39m] Error: Key derivation failed - possibly wrong password
    at Accounts.decrypt (/home/trongnm/Documents/apieth/node_modules/web3-eth-accounts/lib/index.js:438:15)
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:107:53)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[2022-07-07 15:12:03] [[31merror[39m] Error: Validation failed, entered data is incorrect.
    at transfer (/home/trongnm/Documents/apieth/app/controllers/tokenController.js:94:31)
    at Layer.handle [as handle_request] (/home/trongnm/Documents/apieth/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trongnm/Documents/apieth/node_modules/express/lib/router/route.js:144:13)
    at middleware (/home/trongnm/Documents/apieth/node_modules/express-validator/src/middlewares/check.js:16:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
